cmd/skopeo/copy.go

```diff
@@ -72,16 +73,36 @@ See skopeo(1) section "IMAGE NAMES" for the expected format
 	flags.StringSliceVar(&opts.additionalTags, "additional-tag", []string{}, "additional tags (supports docker-archive)")
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Suppress output information when copying images")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Copy all images if SOURCE-IMAGE is a list")
+	flags.StringVar(&opts.listProcessing, "list-processing", "CopySystemImage", "How to process image lists. Choose one of CopySystemImage, CopyAllImages, or CopyNoImages")
```

Comment:
The option name can probably be better.

`list` is not really how users think of multi-arch images; I don’t think `processing` adds any information at all.

Maybe `skopeo copy --multi-arch={system-only|all|none)`? Others can probably suggest better wording.

Reply:
True, I was lost in the API layer. I like your suggestion but I think `none` should be `index-only` - because we're not copying nothing.

Refactored.

Reply:
I am not sure what system (Native arch?) means, but -only is implied.
`--multi-arch={system|all|none)` would make more sense to me.

Reply:
To me, --multi-arch is "what do we do if we're copying a multi-arch (image index) manifest not a standard image.

system = replace it with whatever image from the index matches the running system

all = copy the index and all the architecture

none to me would mean "don't do anything", when actually, the option we want is "copy the index, but don't copy any of the images in the index".

That's why I went with index-only, but happy to change - the doc will explain it clearly anyway.

Thoughts?

Reply:
I agree index-only (metadata-only ? top-level-only? Docker schema2 doesn’t name it “index”) is better than “none”.

I think it’s going to be used fairly rarely, and in unusual situations, so being accurate might be a bit more valuable than being easy to type.